src/packet.cpp

```diff
@@ -28,7 +27,7 @@ Packet::Packet(const std::array<uint8_t, 4096>& m_recvBuffer, size_t bytes, cons
         const uint8_t attributeType = m_recvBuffer[attributeIndex];
         const uint8_t attributeLength = m_recvBuffer[attributeIndex + 1];
 
-        if (attributeType == VENDOR_SPECIFIC)
+        if (attributeType == dict.attributes().code("Vendor-Specific"))
```

Comment:
I think we can keep the original version, with check against VENDOR_SPECIFIC from the enumeration. No need to pull in the Dictionaries.